src/routes/expenses.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const express = require('express');
+const expensesController = require('../controllers/expenses');
+
+const router = express.Router();
+
+router.get('/', express.json(), expensesController.getAll);
+router.get('/:expenseId', express.json(), expensesController.getOne);
+router.post('/', express.json(), expensesController.create);
+router.delete('/:expenseId', express.json(), expensesController.remove);
+router.patch('/:expenseId', express.json(), expensesController.update);
```

Comment:
you can simply simplify this logic, adding Middleware 'express.json()' to app.use -> app.use('/expenses', express.json(), expensesRouter);
```suggestion
router.get('/', expensesController.getAll);
router.get('/:expenseId', expensesController.getOne);
router.post('/', expensesController.create);
router.delete('/:expenseId', expensesController.remove);
router.patch('/:expenseId', expensesController.update);
```
then you should not add it to every request